RELEASE.md

```diff
@@ -0,0 +1,26 @@
+# Release process
+
+This is currently a manual process:
+
+- Bump versions in `web/pf3/package.json` and `web/pf4/package.json`
```

Comment:
Where does the `go` part takes its version?

Reply:
It automatically takes it from the git tag